### ISSUE TITLE
Get voxel value

### ISSUE THREAD

**Issue by yijiang1** [OpenChemistry/tomviz#311]
Is there a way in tomviz to see the value of a particular voxel?

**Comment by mathturtle** [comment]
Maybe.  We don't expose the ParaView feature I'm thinking of.

It depends how you are specifying the voxel.  Are you interested in the value at an index or the value at a point you can see?  You can select points using the small buttons above the view.  Then in ParaView (not tomviz) you can open a spreadsheet view and show the data for the whole volume or just the selection.  Is that the sort of thing you wan to do?

**Comment by yijiang1** [comment]
Yeah, sometimes we want to examine values of particular pixels/voxels in order to understand the dataset better.

> You can select points using the small buttons above the view.

Do you mean this row of buttons? I can't find one that opens a spreadsheet.
<img width="751" alt="screen shot 2015-12-07 at 5 33 41 pm" src="https://cloud.githubusercontent.com/assets/13088588/11642005/8382a4cc-9d09-11e5-8373-169dc62555b0.png">

**Comment by mathturtle** [comment]
Yeah, that row of buttons.  You can select the points (they turn purple by default I think).

If you split the view (the buttons on the right in that image) you can open a spreadsheet view as one of the options, but in tomviz there is no way to load the data into it.

**Comment by yijiang1** [comment]
I see. So currently there is no way for users to get voxel values unless they run a python script?

**Comment by cryos** [comment]
That is correct, it will need to be added to our interface.

**Comment by mathturtle** [comment]
I've been looking at ParaView's Plot Over Line filter to see if we can pull it in to do something like this.  Putting notes here:  Essentially we should be able to use this filter... but we will want to replace the widget used to select the line of interest with one that only selects a subset of our dataset and has options to lock the line parallel to each axis while keeping the ends of the line at the edge of the dataset.
